### ISSUE TITLE
Remote SSH connection "exec: C:WINDOWSSystem32OpenSSHssh.exe: not found" error

### ISSUE THREAD

**Issue by michailj** [microsoft/vscode-remote-release#2612]
After a VS Code update I can't establish a connection with Remote-SSH.
```
[15:50:31.274] Log Level: 2
[15:50:31.274] remote-ssh@0.50.1
[15:50:31.275] win32 x64
[15:50:31.276] SSH Resolver called for "ssh-remote+mj-sandbox", attempt 1
[15:50:31.276] SSH Resolver called for host: mj-sandbox
[15:50:31.276] Setting up SSH remote "mj-sandbox"
[15:50:31.281] Acquiring local install lock: C:\Users\MJAGIE~1\AppData\Local\Temp\vscode-remote-ssh-mj-sandbox-install.lock
[15:50:31.300] Looking for existing server data file at c:\Users\mjagiello4\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-ssh\vscode-ssh-host-mj-sandbox-0ba0ca52957102ca3527cf479571617f0de6ed50-0.50.1\data.json
[15:50:31.302] Using commit id "0ba0ca52957102ca3527cf479571617f0de6ed50" and quality "stable" for server
[15:50:31.304] Install and start server if needed
[15:50:31.308] Checking ssh with "ssh -V"
[15:50:31.659] > OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5
[15:50:31.660] Checking ssh with "C:\WINDOWS\System32\OpenSSH\ssh.exe -V"
[15:50:31.944] > OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5
[15:50:31.944] Checking ssh with "C:\Program Files\Git\usr\bin\ssh.exe -V"
[15:50:32.896] > OpenSSH_8.0p1, OpenSSL 1.1.1c  28 May 2019
[15:50:32.900] Using SSH config file "C:\Users\mjagiello4\.ssh\config"
[15:50:32.901] askpass server listening on \\.\pipe\vscode-ssh-askpass-eee6b7d8e0b1eb9b6fe8055dca60edb96897c867-sock
[15:50:32.904] Spawning local server with {"ipcHandlePath":"\\\\.\\pipe\\vscode-ssh-askpass-b743088ad496eb3a9c7bbae468bdc2be9169ad26-sock","sshCommand":"C:\\Program Files\\Git\\usr\\bin\\ssh.exe","sshArgs":["-v","-T","-D","56404","-F","C:\\Users\\mjagiello4\\.ssh\\config","mj-sandbox"],"dataFilePath":"c:\\Users\\mjagiello4\\AppData\\Roaming\\Code\\User\\globalStorage\\ms-vscode-remote.remote-ssh\\vscode-ssh-host-mj-sandbox-0ba0ca52957102ca3527cf479571617f0de6ed50-0.50.1\\data.json"}
[15:50:32.904] Local server env: {"DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"c:\\Users\\mjagiello4\\.vscode\\extensions\\ms-vscode-remote.remote-ssh-0.50.1\\out\\local-server\\askpass.bat","VSCODE_SSH_ASKPASS_NODE":"C:\\Users\\mjagiello4\\AppData\\Local\\Programs\\Microsoft VS Code\\Code.exe","VSCODE_SSH_ASKPASS_MAIN":"c:\\Users\\mjagiello4\\.vscode\\extensions\\ms-vscode-remote.remote-ssh-0.50.1\\out\\askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"\\\\.\\pipe\\vscode-ssh-askpass-eee6b7d8e0b1eb9b6fe8055dca60edb96897c867-sock"}
[15:50:32.999] Spawned 18648
[15:50:33.231] > local-server> Spawned ssh: 3628
[15:50:33.764] stderr> OpenSSH_8.0p1, OpenSSL 1.1.1c  28 May 2019
[15:50:34.043] stderr> /bin/sh: line 0: exec: C:WINDOWSSystem32OpenSSHssh.exe: not found
[15:50:35.223] stderr> kex_exchange_identification: write: Broken pipe
[15:50:35.229] > local-server> ssh child died, shutting down
[15:50:35.374] Local server exit: 0
[15:50:35.374] Received install output: OpenSSH_8.0p1, OpenSSL 1.1.1c  28 May 2019
/bin/sh: line 0: exec: C:WINDOWSSystem32OpenSSHssh.exe: not found
kex_exchange_identification: write: Broken pipe

[15:50:35.375] Stopped parsing output early. Remaining text: OpenSSH_8.0p1, OpenSSL 1.1.1c  28 May 2019/bin/sh: line 0: exec: C:WINDOWSSystem32OpenSSHssh.exe: not foundkex_exchange_identification: write: Broken pipe
[15:50:35.375] Failed to parse remote port from server output
[15:50:35.377] Resolver error: 
[15:50:35.380] ------
```
Remote - SSH version: 0.50.1
Visual Studio Code version: 1.43.2 (user setup)

**Comment by alberto-giampaolo** [comment]
Hi, I had the same problem after some update or other. I was specifying the location of my ssh.exe in my ssh config file. I was able to fix it by replacing each backslash \ by a double backslash \\.

**Comment by michailj** [comment]
@alberto-giampaolo thanks for response. I have found another solution for that but it's just a workaround. It works for 0.49.0 so it's a regression

**Comment by roblourens** [comment]
This should be fixed in 0.51, please let me know
